test/functional/tests/initialize/test_startup_init_config.py

```diff
@@ -2,24 +2,33 @@
 # Copyright(c) 2019-2021 Intel Corporation
 # SPDX-License-Identifier: BSD-3-Clause-Clear
 #
+import os
+from itertools import cycle
+from time import sleep
 
 import pytest
 
 from api.cas import casadm, casctl, casadm_parser
-from api.cas.casadm_parser import get_caches, get_cores
 from api.cas.cache_config import CacheMode
+from api.cas.casadm import list_caches
+from api.cas.casadm_parser import get_caches, get_cores, get_cas_devices_dict
+from api.cas.cli_messages import check_stdout_msg, no_caches_running
 from api.cas.init_config import InitConfig
 from core.test_run import TestRun
 from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
-from test_utils.filesystem.file import File
+from test_tools.dd import Dd
 from test_tools.disk_utils import Filesystem
+from test_tools.fs_utils import parse_ls_output, readlink, ls
 from test_utils import fstab
-from test_tools.dd import Dd
+from test_utils.filesystem.file import File
+from test_utils.filesystem.symlink import Symlink
+from test_utils.os_utils import drop_caches, sync
 from test_utils.size import Unit, Size
```

Comment:
could use some cleanup, like removing casadm_parser as a whole module from imports (all the needed functions from casadm_parser are imported) or importing casadm and from casadm